app/controllers/api/v1/jwt_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::V1Controller < ApplicationController
+    include JWTHelper
+    skip_before_action :verify_authenticity_token
+
+    def jwt
+        params[:login] = {"login"=>params[:username], "password"=>params[:password], "remember_me"=>"0"}
+        account_or_nil = authenticate(params)
+
+        sign_in(account_or_nil) do |status|
+            if status.success?
+              test = build_jwt(params[:username])
+              render json: test
+            else
+              render json: "Bad login info"
+            end
+          end        
+    end   
+
+    def unsubscribe
+
+        account = decode_jwt(params[:JWT])
```

Comment:
create a before_action method and require that it is called before the method is called